internal/backend/backend_test.go

```diff
@@ -43,16 +43,48 @@ func testForStorageBackends(t *testing.T, test func(t *testing.T, storage Storag
 	}
 }
 
-func noError(t *testing.T, err error) {
+func noError(t *testing.T, err error, description string) {
 	if err != nil {
-		t.Fatal(err)
+		t.Fatalf("%s should not error, but got: %v", description, err)
 	}
 }
 
-func shouldError(t *testing.T, err error, message string) {
+func shouldError(t *testing.T, err error, description string) {
 	if err == nil {
```

Comment:
Should we mark this function as a test helper? This way it won't appear in the stack trace when the test fails.
```suggestion
	t.Helper()
	if err == nil {
```